src/main/java/com/google/devtools/build/lib/remote/RemoteStats.java

```diff
@@ -0,0 +1,49 @@
+package com.google.devtools.build.lib.remote;
+
+import com.google.devtools.build.lib.concurrent.ThreadSafety.ThreadSafe;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/** Collecting always-on stats on a remote run */
+@ThreadSafe
+public class RemoteStats {
+  AtomicInteger total_spawns;
+  AtomicInteger remote_cache_hit;
+  AtomicInteger remote_exec;
+  AtomicInteger local;
+
+  public RemoteStats() {
+    total_spawns = new AtomicInteger();
+    remote_cache_hit = new AtomicInteger();
+    remote_exec = new AtomicInteger();
+    local = new AtomicInteger();
+  }
+
+  public void addSpawn() {
```

Comment:
`add` doesn't seem like a great fit ... name it `countSpawn`? Same for the methods below.